aws-sso-permissionset/aws-sso-permissionset.json

```diff
@@ -0,0 +1,160 @@
+{
+  "typeName": "AWS::SSO::PermissionSet",
+  "description": "Resource Type definition for SSO PermissionSet",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-sso/aws-sso-permissionset",
+  "definitions": {
+    "Tag": {
+      "description": "The metadata that you apply to the permission set to help you categorize and organize them.",
+      "type": "object",
+      "properties": {
+        "Key": {
+          "type": "string",
+          "pattern": "[\\w+=,.@-]+",
+          "minLength": 1,
+          "maxLength": 128
+        },
+        "Value": {
+          "type": "string",
+          "pattern": "[\\w+=,.@-]+",
+          "minLength": 0,
+          "maxLength": 256
+        }
+      },
+      "required": [
+        "Key",
+        "Value"
+      ]
+    },
+    "ManagedPolicyArn": {
+      "description": "The managed policy to attach.",
+      "type": "string",
+      "minLength": 20,
+      "maxLength": 2048
+    }
+  },
+  "properties": {
+    "Name": {
+      "description": "The name you want to assign to this permission set.",
+      "type": "string",
+      "pattern": "[\\w+=,.@-]+",
+      "minLength": 1,
+      "maxLength": 32
+    },
+    "PermissionSetArn": {
+      "description": "The permission set that the policy will be attached to",
+      "type": "string",
+      "pattern": "arn:aws:sso:::permissionSet/(sso)?ins-[a-zA-Z0-9-.]{16}/ps-[a-zA-Z0-9-./]{16}",
```

Comment:
[should un-hardcode the `aws` partition](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arns-syntax) eventually in order to work in [other partitions](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants)